editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -42,58 +45,70 @@ export const MembersTable = ({
   }
 
   return (
-    <TableContainer>
-      <Table>
-        <TableHead>
-          <StyledTableRow>
-            <TableCell sx={{ width: 300 }}>
-              <strong>User</strong>
-            </TableCell>
-            <TableCell sx={{ width: 200 }}>
-              <strong>Role</strong>
-            </TableCell>
-            <TableCell>
-              <strong>Email</strong>
-            </TableCell>
-          </StyledTableRow>
-        </TableHead>
-        <TableBody>
-          {members.map((member) => (
-            <StyledTableRow key={member.id}>
-              <TableCell
-                sx={{
-                  display: "flex",
-                  flexDirection: "row",
-                  alignItems: "center",
-                }}
-              >
-                <StyledAvatar>
-                  {member.firstName[0]}
-                  {member.lastName[0]}
-                </StyledAvatar>
-                {member.firstName} {member.lastName}
+    <>
+      <TableContainer>
+        <Table>
+          <TableHead>
+            <StyledTableRow>
+              <TableCell sx={{ width: 300 }}>
+                <strong>User</strong>
+              </TableCell>
+              <TableCell sx={{ width: 200 }}>
+                <strong>Role</strong>
               </TableCell>
               <TableCell>
-                <Chip
-                  label={getRoleLabel(member.role)}
-                  size="small"
-                  sx={{ background: "#ddd" }}
-                />
+                <strong>Email</strong>
               </TableCell>
-              <TableCell>{member.email}</TableCell>
             </StyledTableRow>
-          ))}
-          {showAddMemberButton && hasFeatureFlag("ADD_NEW_EDITOR") && (
-            <TableRow>
-              <TableCell colSpan={3}>
-                <AddButton onClick={() => setShowModal(true)}>
-                  Add a new editor
-                </AddButton>
-              </TableCell>
-            </TableRow>
-          )}
-        </TableBody>
-      </Table>
-    </TableContainer>
+          </TableHead>
+          <TableBody
+            data-testid={`members-table${showAddMemberButton && "-add-editor"}`}
+          >
+            {tableData.map((member) => (
```

Comment:
This looks like a lot of changes but it's literally just adding a testId to the members table if it's the one with the 'add editor' button 😅